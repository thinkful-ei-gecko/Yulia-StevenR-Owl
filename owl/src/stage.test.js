import React from 'react';
import ReactDOM from 'react-dom';
import Stage from './stage';
import participants from './Store';
import renderer from 'react-test-renderer';


describe('stage', () => {
  it('renders without crashing', () => {
    // first create a DOM element to render the component into
    const div = document.createElement('div');
  
    //render the component, this is the actual test, if something is wrong it will fail here
    ReactDOM.render(<Stage participants={participants}/>, div);
  
    //clean up code
    ReactDOM.unmountComponentAtNode(div);
  });
  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Stage participants={participants}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });
});