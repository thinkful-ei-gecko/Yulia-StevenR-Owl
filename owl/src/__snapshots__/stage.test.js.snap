// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stage renders the UI as expected 1`] = `
<section>
  <span
    className="square"
  >
    <h1>
      Koren Templeton
    </h1>
    <img
      alt="participantsAvatar"
      src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
    />
  </span>
  <span />
  <span />
  <span />
  <span
    className="square"
  >
    <h1>
      Ashla Attwool
    </h1>
    <img
      alt="participantsAvatar"
      src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
    />
  </span>
</section>
`;
